portfolio/src/main/webapp/index.html

```diff
@@ -74,10 +74,13 @@ <h1>Life Before Quarantine</h1>
         </div>
       </div>
     </section>
-    <section id="comments">
-      <button type="button" class="btn btn-outline-primary" 
-      onclick="getComments()">See Comments</button>
+    <section id="comments" >
       <ul id="comments-grid"></ul>
+      <form action="/data" method="POST">
+        <label for="comment-box">Drop a Comment!</label><br>
+        <textarea name="text-input" id="comment-box"></textarea><br>
+        <input type="submit" />
```

Comment:
Quick nit here: since you're in an HTML5 context, you don't need the trailing "/" in self-closing tags.